### PR TITLE
Updated missing section about exclude-multi-locale

### PR DESCRIPTION
**Pull request by Delphine**
This tackles what was missing for this doc about "exclude-multi-locale" section of toml file, as per #121

### CODE REVIEW

**Comment by Delphine** [comment]
Realized that the multi-locales doc needed to be updated as well in the meantime :)

**Comment by Delphine** [comment]
Hmm on second thoughts, I'm thinking I'm not explaining things correctly in the multi-locale doc.
Is it that you only remove it from the "exclude-multi-locale" section once you've added the locale to multi-locales and done a sign-off so that it is now on Beta? I'm not quite sure how this is supposed to work, not having done it before :/ thanks for helping clarify that part!

File: products/firefox_android/adding_singlelocale.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ locales = [
 
 Identify the `locales` section, and add the new locale code between double quotes, followed by a comma. As before, you can use Atom to make sure the list is in alphabetical order (make sure to select only the lines with actual locale codes before pressing `F5`).
 
+Once this is done, identify the `exclude-multi-locale` section. Since this locale is not shipping on multi-locale builds yet, you should add it there too, in the same way you just did above.
```
Comment (flodolo) [inline products/firefox_android/adding_singlelocale.md]: Let's remove the "in the same way…"?

File: products/firefox_android/adding_multilocales.md [hunk 2]
```diff
@@ -28,15 +28,38 @@ $ hg pull -r default -u
...
+    "ab-CD",
+    "an",
+    "ar",
+    "as",
+    "ast",
+    "az",
+...
+```
+
+Identify the `locales` section, and make sure the locale is already listed under the `locales` section (it should be already). In the example above, the locale is represented by `ab-CD`.
```
Comment (flodolo) [inline products/firefox_android/adding_multilocales.md]: Maybe: it should be already -> it should have been added when enabling single locale builds?

File: products/firefox_android/adding_multilocales.md [hunk 3]
```diff
@@ -28,15 +28,38 @@ $ hg pull -r default -u
...
+
+Then locate the `exclude-multi-locale` section, and remove the locale from there.
```
Comment (flodolo) [inline products/firefox_android/adding_multilocales.md]: from there -> from this list

**Review by flodolo — COMMENTED** [review]
Some minor nits but it looks pretty clear to me.

**Comment by flodolo** [comment]
> Is it that you only remove it from the "exclude-multi-locale" section once you've added the locale to multi-locales and done a sign-off so that it is now on Beta? I'm not quite sure how this is supposed to work, not having done it before :/ thanks for helping clarify that part!

Right now it doesn't have a practical effect, but it's correct as it is right now: you add the file to maemo-locales and remove it from the excluded list at the same time, since these two operations have the same goal (add locale to multi-locales build).

For both, you need a sign-off on Beta before it goes to build.

File: products/firefox_android/adding_multilocales.md (hunk 2 above)
Comment (Delphine) [reply products/firefox_android/adding_multilocales.md]: Agreed, sounds better.

File: products/firefox_android/adding_multilocales.md (hunk 3 above)
Comment (Delphine) [reply products/firefox_android/adding_multilocales.md]: 👍

File: products/firefox_android/adding_singlelocale.md (hunk 1 above)
Comment (Delphine) [reply products/firefox_android/adding_singlelocale.md]: Yeah it's a "heavy" sentence, agreed we can drop that

**Review by flodolo — APPROVED** [review]

**Pull request merged**